File: problems/2714-left-and-right-sum-differences/left-and-right-sum-differences.java
class Solution {
    public int[] leftRightDifference(int[] nums) {
        int l[]=new int[nums.length];
        for (int i=0;i<nums.length;i++) {
            int ls=0,rs=0;
            for (int j=0;j<i;j++)   ls+=nums[j];
            for (int j=i+1;j<nums.length;j++)  rs+=nums[j];
            l[i]=(ls>rs)?ls-rs:rs-ls;
        }
        return l;
    }
}

File: problems/2714-left-and-right-sum-differences/left-and-right-sum-differences.py
class Solution(object):
    def leftRightDifference(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        l=[]
        for i in range (0,len(nums)):
            ls,rs=0,0
            for j in range(0,i):
                ls+=nums[j]
            for j in range(i+1,len(nums)):
                rs+=nums[j]
            l.append(max(ls,rs)-min(ls,rs))
        return l